packages/playwright-test/src/matchers/matchers.ts

```diff
@@ -309,3 +311,51 @@ export async function toBeOK(
   const pass = response.ok();
   return { message, pass };
 }
+
+export async function toPass(
+  this: ReturnType<Expect['getState']>,
+  callback: () => Promise<void>,
+  options: {
+    intervals?: number[];
```

Comment:
Missing custom `message`?

Reply:
Custom messages are handled on the higher level.